x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -65,14 +65,19 @@ import {
 import { SecurityAppStore } from './common/store/store';
 import { getCaseConnectorUI } from './cases/components/connectors';
 import { licenseService } from './common/hooks/use_license';
+import { SecuritySolutionConfigType } from './common/types';
```

Comment:
(optional) consider renaming so that its clear this is only a subset of the security soluton configuration (the properties exposed to the UI). Maybe `SecuritySolutionUiConfigType`?